hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassDefinitionsServiceImpl.java

```diff
@@ -95,6 +95,9 @@
     private final MsgLogger msgLog = MsgLogger.LOGGER;
     private final Logger log = Logger.getLogger(CassDefinitionsServiceImpl.class);
 
+    private Set<DefinitionsEvent> deferredNotifications = new HashSet<>();
```

Comment:
The idea was not to carry state between multiple calls into the EJB, but to provide a deferNotification status through one call to import (or, feasibly, any single EJB method that wanted to perform multiple updates).   So, I think this approach is OK.